src/plasoscaffolder/frontend/controller/sqlite_controller.py

```diff
@@ -204,59 +175,64 @@ def _CreateSQLQueryModelWithUserInput(
       query: str, with_examples: bool,
       query_execution: base_sql_query_execution.BaseSQLQueryExecution()
   ) -> sql_query_model.SQLQueryModel:
-    """Asks the user information about the sql query
+    """Asks the user information about the SQL Query
   
     Args:
-      query (str): the sql query
-      with_examples (bool): if the user wants examples for the given query
+      query (str): the SQL Query
+      with_examples (bool): if the user wants examples for the given Query
   
     Returns:
-      (sql_query_model.SQLQueryModel) a sql query model
+      sql_query_model.SQLQueryModel: a SQL Query model
     """
     query_data = self._plugin_helper.RunSQLQuery(query, query_execution)
 
     if query_data.has_error:
-      self._output_handler.PrintError('The SQLQuery has an Error.')
       self._output_handler.PrintError(str(query_data.error_message))
       return None
 
     else:
       if with_examples:
         length = len(query_data.data)
-        if length == 0 :
+        if length == 0:
           self._output_handler.PrintInfo('Your query does not return anything.')
         else:
-          first_line = '\n{0}'.format(query_data.data[0]) if 1 <= length else ''
-          second_line = '\n{0}'.format(query_data.data[1]) if 2 <= length else ''
-          third_line = '\n{0}'.format(query_data.data[2]) if 3 <= length else ''
           self._output_handler.PrintInfo(
-              'Your query output could look like this.{0}{1}{2}'.format(
-                  first_line, second_line, third_line))
+              'Your query output could look like this.')
+          self._output_handler.PrintInfo(
+              str(list(map(lambda x: x.SQLColumn, query_data.columns))))
+          if length < self.AMOUNT_OF_SQLITE_OUTPUT_EXAMPLE:
+            amount = length
+          else:
+            amount = self.AMOUNT_OF_SQLITE_OUTPUT_EXAMPLE
+          for i in range(0, amount):
+            self._output_handler.PrintInfo(str(query_data.data[i]))
+
         add_query = self._output_handler.Confirm(
             'Do you want to add this query?',
             abort=False, default=True)
         if not add_query:
           return None
+      else:
+        self._output_handler.PrintError('The SQL query was ok.')
 
-      message = 'What kind of row does the SQL query parse?'
-      name = self._output_handler.PromptInfo(text=message)
-      whole_name = 'Parse{0}Row'.format(name.title())
+      name = ''.join(explain_query_plan.ExplainQueryPlan(
+          query_execution).getLockedTables(query)).capitalize()
```

Comment:
broke it up to:
```
query_plan = explain_query_plan.ExplainQueryPlan(query_execution)
locked_tables = query_plan.getLockedTables(query)
name = ''.join(locked_tables).capitalize()
```
I guess it is enough. 
Or do you want to break up the join and capitalize as well?